bin/unpack-node

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+
+mkdir -p node
+pushd node
+
+config=`../bin/config.guess`
```

Comment:
Maybe check Python's `sys.platform` instead of bundling config.guess?

Reply:
This comes back to the question of whether we want/need to support non-Docker local development (or at least node/yarn version-specific local dev). If not, this can just be removed.

Reply:
I think we can remove this, given today's decisions.

Reply:
@wlonk This will still be used by CI to ensure the right node/yarn versions, unless/until we switch to Docker there.

Reply:
OK 👍